### DISCUSSION TITLE
ListSuggester show PDF File links?

### DISCUSSION THREAD

**Discussion by xverbatim1971** [mProjectsCode/obsidian-meta-bind-plugin#155] [General]
hello,

first of all, many thanks to the developer. this is such a useful plugin. thank you very much!

one question, in the listsuggester, is there a possibility to select pdf files as well? Or is such a function planned?

that would be so great to have files that are not md notes in the listsuggester and pdf would be extremely useful :-)

maybe there is another possibility? 

 ```meta-bind
INPUT[listSuggester(optionQuery("Folder Test"), showcase):listtest]
```

**Comment by mProjectsCode** [comment]
I am using dataview to get the filed displayed there, so any FR regarding that has to be directed at dataview.

**Reply by xverbatim1971** [reply]
hello,
yes, that would be a possibility, but it would be rather inconvenient for me.

I have a main folder (customers) and a folder for invoices and reminders... 

So I would have the possibility with the listsuggester to assign the files to the customers.

I would have to create a folder for each customer via Dataview and I don't want to do that because I would then always have to keep an eye on where invoices are saved ... chaos :(

**Reply by mProjectsCode** [reply]
what i mean is that if you want other file types to be recognized by the suggester, they need to be recognized by dataview, since the plugin uses dataview to get the list of files to display.

**Reply by xverbatim1971** [reply]
oh okay, I didn't know that.

that's difficult then ... 

too bad, i would have liked a function that allows me to easily import pdf links with preview via a button, that would have been perfect ...
